feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.html

```diff
@@ -4,23 +4,45 @@
     role="search"
     [attr.aria-label]="'quickOrderForm.quickOrderSearch' | cxTranslate"
   >
-    <input
-      (blur)="onBlur($event)"
-      (focus)="open()"
-      (keydown.arrowdown)="focusNextChild($event)"
-      (keydown.arrowup)="focusPreviousChild($event)"
-      (keydown.enter)="addProduct($event)"
-      (keydown.escape)="clear($event)"
-      [attr.aria-label]="
-        'quickOrderForm.searchBoxLabel' | cxTranslate: { limit: limit }
-      "
-      aria-controls="results"
-      aria-describedby="quickOrderFormInitialDescription"
-      class="form-control"
-      formControlName="product"
-      placeholder="{{ 'quickOrderForm.placeholder' | cxTranslate }}"
-      type="text"
-    />
+    <!-- TODO:(CXSPA-1695) for next major release update below feature level check, we renamed aria-controls-->
```

Comment:
I currently do not see quick-order-form.component.html in the ticket.

Reply:
added thanks~